### ISSUE TITLE
Incoming message don't work with bolean

### ISSUE THREAD

**Issue by emmanuel-landrivon** [bartbutenaers/node-red-contrib-ui-multistate-switch#17]
Hi,
Thanks so much for this dev.

Just report small issue : the remotly change doesn't work with bolean (work with string)

Thanks a lot
Regards

`[
    {
        "id": "3eb942ef8ccb43ba",
        "type": "ui_multistate_switch",
        "z": "19a2272fa7ee0c60",
        "name": "",
        "group": "074c0fe00af791b1",
        "order": 0,
        "width": 0,
        "height": 1,
        "label": "test",
        "stateField": "payload",
        "enableField": "enable",
        "passthroughField": "passthrough",
        "inputMsgField": "inputmsg",
        "rounded": true,
        "useThemeColors": true,
        "hideSelectedLabel": false,
        "multilineLabel": false,
        "passThrough": "always",
        "inputMsg": "all",
        "userInput": "enabled_show",
        "options": [
            {
                "label": "Open",
                "value": "true",
                "valueType": "bool",
                "color": "#009933"
            },
            {
                "label": "Close",
                "value": "false",
                "valueType": "bool",
                "color": "#999999"
            }
        ],
        "x": 850,
        "y": 1740,
        "wires": [
            [
                "7ff76787dd4aa69b"
            ]
        ]
    },
    {
        "id": "6d362bc0b040441a",
        "type": "inject",
        "z": "19a2272fa7ee0c60",
        "name": "",
        "props": [
            {
                "p": "payload"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "true",
        "payloadType": "bool",
        "x": 710,
        "y": 1720,
        "wires": [
            [
                "3eb942ef8ccb43ba"
            ]
        ]
    },
    {
        "id": "4e0c1713807f8bfc",
        "type": "inject",
        "z": "19a2272fa7ee0c60",
        "name": "",
        "props": [
            {
                "p": "payload"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "false",
        "payloadType": "bool",
        "x": 710,
        "y": 1760,
        "wires": [
            [
                "3eb942ef8ccb43ba"
            ]
        ]
    },
    {
        "id": "7ff76787dd4aa69b",
        "type": "debug",
        "z": "19a2272fa7ee0c60",
        "name": "",
        "active": true,
        "tosidebar": true,
        "console": false,
        "tostatus": false,
        "complete": "false",
        "statusVal": "",
        "statusType": "auto",
        "x": 1050,
        "y": 1740,
        "wires": []
    },
    {
        "id": "074c0fe00af791b1",
        "type": "ui_group",
        "name": "Reactor",
        "tab": "417f890de8cd16e6",
        "order": 1,
        "disp": true,
        "width": "16",
        "collapse": false,
        "className": ""
    },
    {
        "id": "417f890de8cd16e6",
        "type": "ui_tab",
        "name": "Home",
        "icon": "dashboard",
        "disabled": false,
        "hidden": false
    }
]`

**Comment by bartbutenaers** [comment]
Hey @emmanuel-landrivon,

Thanks for reporting this bug!
It should be fixed now on Github (not published on NPM yet...).
If you like, you can test it by installing this node directly from Github (from within your .node-red folder):
```
npm install bartbutenaers/node-red-contrib-ui-multistate-switch
```
 Bart

**Comment by bartbutenaers** [comment]
This fix is now available as version ***1.2.2*** in the palette:

![image](https://user-images.githubusercontent.com/14224149/142074489-3f3a9b78-ecc5-4134-b73c-02ea52a071f7.png)
